app/scripts/directives/seekBar.js

```diff
@@ -30,6 +30,10 @@
                     return {width: percentString()};
                 };
                 
+                scope.thumbStyle = function(){
+                    return {width: 10};
```

Comment:
So this sets the thumbnail to always have a width 10 at all times.. but it stays in place.  Don't we want the thumb to move around as you drag your mouse?